cli/tests/services.bats

```diff
@@ -91,6 +91,8 @@ setup() {
 }
 
 teardown() {
+  # Ensure the watchdog has a chance to clean up before deleting the test directory
+  sleep 0.1
```

Comment:
Can we assume cleanup happens in bounded time?
Also, why do we need to block the teardown anyway, are tests reusing the same resources?

Reply:
Dropped in https://github.com/flox/flox/pull/2018/commits/e04ad17546f72830ad7c74c59129aad32b99a904

Reply:
And added polling in https://github.com/flox/flox/pull/2018/commits/0d99bc4c753a68bc1ab2e9bff17a953073fe9cf0